public/docs/ts/latest/guide/architecture.jade

```diff
@@ -1,190 +1,218 @@
   and we'll talk about components in the next segment. For the moment it is enough to know that a
   component class is the kind of thing we'd export from a module.
-  
-  Most applications have an `AppComponent`. By convention, we'll find it in a file named `app.component.ts`. 
-  Look inside such a file and we'll see an `export` statement like this one.
-+makeExample('architecture/ts/app/app.component.ts', 'export', 'app/app.component.ts (excerpt)')(format=".")
+
+  Most applications have an `AppComponent`. By convention, we'll find it in a file named `app.component.ts`.
+  Look inside such a file and we'll see a declaration like this one.
+
++makeExcerpt('app/app.component.ts ()', 'export')
+
+block ts-export
+  :marked
+    The `export` statement tells TypeScript that this is a module whose
+    `AppComponent` class is public and accessible to other modules of the application.
+
 :marked
-  The `export` statement tells TypeScript that this is a module whose
-  `AppComponent` class is public and accessible to other modules of the application.
-  
   When we need a reference to the `AppComponent`, we **import** it like this:
-+makeExample('architecture/ts/app/main.ts', 'import', 'app/main.ts (excerpt)')(format=".")
+
++makeExcerpt('app/main.ts', 'import')
+
+block ts-import
+  :marked
+    The `import` statement tells the system it can get an `AppComponent` from a module named `app.component`
+    located in a neighboring file.
+    The **module name** (AKA module id) is often the same as the filename without its extension.
+
 :marked
-  The `import` statement tells the system it can get an `AppComponent` from a module named `app.component`
-  located in a neighboring file. 
-  The **module name** (AKA module id) is often the same as the filename without its extension.
-  ### Library Modules 
+  ### Libraries
+
 figure
   img(src="/resources/images/devguide/architecture/library-module.png" alt="Component" align="left" style="width:240px; margin-left:-40px;margin-right:10px" )
+
+block angular-library-modules
+  :marked
+    Some modules are _libraries_ of other modules.
+    Angular itself ships as a collection of library modules within several npm packages.
+    Their names begin with the `!{_at_angular}` prefix.
+
+    Each Angular library contains a [barrel](../glossary.html#barrel) module
+    that is actually a public façade over several logically-related private modules.
+
 :marked
-  Some modules are libraries of other modules.
-  
-  Angular itself ships as a collection of library modules within several npm packages.
-  Their names begin with the `@angular` prefix. 
-  Each Angular library contains a [barrel](../glossary.html#barrel) module
-  that is actually a public façade over several logically-related private modules.
-  
-  The `@angular/core` library is the primary Angular library module from which we get most of what we need.
+  `!{_at_angular}/core` is the primary Angular library from which we get most of what we need.
```

Comment:
This discussion of specific libraries seems like the most useful part of this section.

I'm just not sure whether the concept of "modules" is something we need for the Dart side.